django/utils/text.py

```diff
@@ -322,7 +322,7 @@ def close(self):
 # Like compress_string, but for iterators of strings.
 def compress_sequence(sequence):
     buf = StreamingBuffer()
-    with GzipFile(mode='wb', compresslevel=6, fileobj=buf) as zfile:
+    with GzipFile(mode='wb', compresslevel=6, fileobj=buf, mtime=0) as zfile:
```

Comment:
The modification time doesn't matter for streaming responses (which is what uses this method), since the purpose is to allow ETag matches and streaming responses don't generate ETags. So I don't think it needs a test. Since it doesn't matter, I prefer the `mtime=0` for consistency (people might wonder why they're different).